client/lib/posts/post-list-store-factory.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Internal Dependencies
+ **/
+import PostListStore from './post-list-store';
+
+/**
+ * Module variables
+ **/
+const _postListStores = {};
+
+export default function( storeId ) {
+	const postStoreId = storeId || 'default';
+	let postListStore = _postListStores[ postStoreId ];
```

Comment:
Not necessary to change, but in these cases, I find it a bit more concise and easy-to-follow to ensure the object contains what I'm looking for, rather than shortcutting early in the case that it happens to already contain it.

```js
export default function( storeId ) {
	const postStoreId = storeId || 'default';

	if ( ! _postListStores[ postStoreId ] ) {
		_postListStores[ postStoreId ] = new PostListStore( postStoreId );
	}

	return _postListStores[ postStoreId ];
}
```